packages/project-disaster-trail/src/components/Game/TaskScreen/SolveScreen.js

```diff
@@ -82,6 +83,15 @@ class SolveScreen extends PureComponent {
       background: beige;
     `;
 
+    const taskImage = css`
+      background-image: url(${activeTask.imageSVG});
+      background-repeat: no-repeat;
+      background-size: cover;
+      margin: 15px;
+      height: 150px;
+      width: 150px;
+    `;
```

Comment:
I'm not sure what we're doing pixel-wise in other places. The original plan for this game was that it would size dynamically for at least laptop and touchscreen size. @lucastswick, do you have any comments here?